wide-graph-with-submodules/README.md

```diff
@@ -0,0 +1,11 @@
+# Wide graph with submodules
+
+The test in this directory exists to simulate a wide graph scenario with lots of submodules to expand.
+
+## Used to validate:
+
+### ModuleExpansionTransformer performance https://github.com/opentofu/opentofu/pull/1809
+Apple M2 Pro / 32GB RAM
+
+With 10x1000 modules, this dropped from 34s to 11s.
```

Comment:
What do you mean 10x1000?

Reply:
In `main.tf` I have hard-coded 10 top-level modules that each have `source = "./generated"`, which is 1000 instances of `source = "../submodule"` each, so that feels like 10x1000 to me, but if there's a more eloquent way to describe this I'm happy to update it.